packages/react-native-reanimated/Common/cpp/worklets/NativeModules/NativeWorkletsModule.cpp

```diff
@@ -0,0 +1,27 @@
+#include <string>
+
+#ifdef RCT_NEW_ARCH_ENABLED
+#include <react/renderer/uimanager/UIManagerBinding.h>
+#include <react/renderer/uimanager/primitives.h>
+#endif // RCT_NEW_ARCH_ENABLED
+#include <worklets/NativeModules/NativeWorkletsModule.h>
+#ifdef __ANDROID__
+#include <fbjni/fbjni.h>
+#endif // __ANDROID__
+
+#include <jsi/jsi.h>
+
+using namespace facebook;
+
+namespace reanimated {
+
+NativeWorkletsModule::NativeWorkletsModule(const std::string &valueUnpackerCode)
+    : NativeWorkletsModuleSpec(nullptr),
```

Comment:
This `nullptr` looks a bit suspicious 👀

Reply:
It's okay for now since Worklets don't emit any events or anything or the sort. In later PRs we'll forward the JSCallInvoker here on Paper.